### ISSUE TITLE
Cordova SQLite DATE conditioned query

### ISSUE THREAD

**Issue by mareksip** [brody2consult/cordova-sqlite-storage#415]
I have Receipt table that I am trying to SELECT data from by conditions in DATE datatype column. 

The format in Browser (WebSQL) like this:

DateIssue DATE format: `Thu Feb 04 2016 23:09:15 GMT+0100 (Střední Evropa (běžný čas))`

My query:

```
tx.executeSql('SELECT ri.ProductId, ri.Name as riName FROM ReceiptItem ri INNER JOIN
Receipt r ON r.ID = ri.ReceiptId WHERE   
r.DateIssue >= ? AND r.DateIssue <= ? GROUP BY ri.ProductId', [dateFrom, dateTo], function..
```

[dateFrom]
`Mon Feb 01 2016 23:59:59 GMT+0100 (Střední Evropa (běžný čas))`

[dateTo]
`Mon Feb 29 2016 23:59:59 GMT+0100 (Střední Evropa (běžný čas))`

No records get selected. Is SQLite supporting standard JS date data type?

This is how I parse the input date:

```
//Parse TO DATE
var dTo = $scope.data.dateTo;
dTo.setHours(23, 59, 59);
var toParsed = Date.parse(dTo);
var dateToFull = new Date(toParsed);
```

**Comment by brody2consult** [comment]
I just saw your report. The plugin does not explicitly deal with the Javascript Date object.

As a workaround you _should_ be able to use built-in sqlite functions to deal with the conversions. Here are a few resources I found:
- http://stackoverflow.com/questions/18244979/storing-a-datetime-value-in-web-sql
- http://stackoverflow.com/questions/10104893/format-date-in-websql
- http://www.sqlite.org/lang_datefunc.html

Thanks for reporting. I will investigate this when I get a chance.
